meson.build

```diff
@@ -74,17 +75,35 @@ if get_option('wayland') != 'false'
   endif
 endif
 
+# WIN32
+use_win32 = false
+if get_option('win32') != 'false'
+  require_win32 = get_option('win32') == 'true'
+  use_win32 = dependency('libva-win32', required: require_win32).found()
+  if use_win32
+    libva_utils_flags = [ '-DHAVE_VA_WIN32=1', '-DNOMINMAX']
+  endif
+endif
+
 add_project_arguments(libva_utils_flags,
                       language: ['c', 'cpp'])
 
-subdir('common')
-subdir('decode')
-subdir('encode')
-subdir('putsurface')
+if use_win32
+  subdir('getopt')
+  win32_deps = [ dependency('libva-win32', required: require_win32), idep_getopt ]
+endif
+
+subdir('common') # Uses win32_deps
 subdir('vainfo')
-subdir('videoprocess')
-subdir('vendor/intel')
-subdir('vendor/intel/sfcsample')
+
+if not use_win32
+  subdir('decode')
```

Comment:
is it possible to make sample codes work on windows?

Reply:
For most of them we don't have driver support yet to test it (ie. mpeg2, jpeg decode, vp8/vp9 encode and most VPP effects tested). I think the ones we could port and test today are: encode H264/HEVC and VPP scaling and color conversion (only NV12, P010 and RGBX supported in driver at the moment). 

From those we could port and test at the moment, cross-platform porting is needed, the most notable port needed would be `pthreads` for the encode samples. 

As we're also in the process of writing new samples for Windows that cover those same Decode/Encode/VPP scenarios implemented by the driver today, also showing D3D12/Libva interoperability and adapter enumeration/selection, could we maybe point to those samples instead in the documentation? The goal is to upstream the open source samples to [DirectX-Graphics-Samples](https://github.com/microsoft/DirectX-Graphics-Samples) when finished.

You can find the in-progress samples in: [H264 decode sample](https://github.com/sivileri/DirectX-Graphics-Samples/tree/video_sample/Samples/Desktop/D3D12HelloWorld/src/HelloVADecode) similar to `intel/sfcsample` and [VPP Color Conversion + H264 encode](https://github.com/sivileri/DirectX-Graphics-Samples/tree/video_sample/Samples/Desktop/D3D12HelloWorld/src/HelloVAEncode) that show `encode` and `color conversion`